src/utils/templates.ts

```diff
@@ -66,7 +66,7 @@ export const updatePackageJson = (
 export const displayFinalInstructions = (appName: string) => {
   console.log(chalk.green(`🚀 Project '${appName}' created successfully!`));
   console.log(
-    chalk.blue(`Type 'cd ${appName}' to navigate into your new project.`)
+    chalk.blue(`Type 'cd apps/${appName}' to navigate into your new project.`)
```

Comment:
Ops, forgot this one. In the next PR I am going to have the `apps` folder committed and the code need it for this.